.github/workflows/scala.yml

```diff
@@ -0,0 +1,16 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: olafurpg/setup-scala@v2
+        with:
+          java-version: 1.8
+      - name: Run compile
+        run: csbt +compile
```

Comment:
Why not use `csbt +test` here?

Reply:
There aren't tests on this package

Reply:
I was thinking that in this case `test` will do the same as `compile`, but we won't have to change it if we ever implement tests.